packages/fdc3-agent-proxy/src/channels/DefaultChannel.ts

```diff
@@ -98,4 +102,30 @@ export class DefaultChannel implements Channel {
     await listener.register();
     return listener;
   }
+
+  async clearContext(contextType?: string): Promise<void> {
+    // first, ensure channel state is up-to-date
+    const request: ClearContextRequest = {
+      meta: this.messaging.createMeta(),
+      payload: {
```

Comment:
Also, there will need to be support for this event added to the `BroadcastHandler` in the `fdc3-web-impl`, plus tests of the same there.